packages/@react-aria/datepicker/intl/ar-AE.json

```diff
@@ -1,17 +1,18 @@
 {
   "calendar": "التقويم",
-  "day": "يوم",
-  "dayPeriod": "ص/م",
+  "day": "اليوم",
```

Comment:
According to google translate, this changed from the arabic word for "day" to "today", which seems different. Similar changes for some other strings here.

For context, these existing translations for field names (e.g. year, month, day, hour, etc.) were imported from Unicode CLDR (we are polyfilling the `Intl.DisplayNames` API for some browsers). I figured you wouldn't need to re-translate them. Probably the translators were missing some context about where these would be used. Would you recommend we store those externally to these JSON files so they aren't picked up by automation? Should we revert those strings so that they match CLDR (and thus, the native browser API where available)? Sorry this probably ended up being a bunch of extra work...

Reply:
@devongovett this Is a bit tricky, as translation automation will end up replacing any existing translation from GIT with translations from our backend 'glaas' database, with is what translator uses. Any direct GIT translation would eventually get replaced, and up to that translator.

CLDR is the perfect standard, let me check to update our backend to use the original translation, which would fix this issue for now (and revert back a number of changes), I'll check and get back to you. Translations should not change again, unless someone reports a JIRA requesting the translation be changed, then translator would update glass.

CLDR does have translation files that can be used and may be better long term solution, use those and not pass through our system, for now through I'll check and revert ones I can, I'll give update later today or morning